lib/generators/doorkeeper/templates/initializer.rb

```diff
@@ -313,6 +313,16 @@
   #
   # handle_auth_errors :raise
 
+  # If an exception occurs during the authorization request, Doorkeeper will, by
+  # default, render an HTML error response with the exception message and HTTP
+  # status code. If you want to redirect back to the client application in accordance
+  # with https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1, you can
+  # enable this option.
+  #
+  # This will have no effect if handle_auth_errors is set to :raise.
+  #
+  # redirect_on_error
```

Comment:
Or maybe this should just be configured as `handle_auth_errors :redirect`?

Reply:
Yeah, I went ahead and did this since I think it makes more sense and it cleaner this way 👍